src/content/docs/identityserver/overview/packaging.mdx

```diff
@@ -37,26 +37,240 @@ dotnet new -i Duende.Templates
 ```
 
 <CardGrid>
-  <LinkCard
-    description="NuGet Package for IdentityServer Templates"
-    href="https://www.nuget.org/packages/Duende.Templates"
-    title="Templates"
-    target="_blank"
-  />
-  <LinkCard
-    description="Source code for IdentityServer Templates"
-    href="https://github.com/DuendeSoftware/IdentityServer.Templates"
-    title="Source Code"
-  />
+    <LinkCard
+        description="NuGet Package for IdentityServer Templates"
+        href="https://www.nuget.org/packages/Duende.Templates"
+        title="Templates"
+        target="_blank"
+    />
+    <LinkCard
+        description="Source code for IdentityServer Templates"
+        href="https://github.com/DuendeSoftware/IdentityServer.Templates"
+        title="Source Code"
+    />
 </CardGrid>
 
-## UI
+Running the command `dotnet new list duende` should give you a list of the following templates
+
+```bash
+Template Name                                               Short Name            Language  Tags
+----------------------------------------------------------  --------------------  --------  -------------------------
+Duende BFF Host using a Remote API                          duende-bff-remoteapi  [C#]      Web/Duende/BFF
+Duende BFF using a Local API                                duende-bff-localapi   [C#]      Web/Duende/BFF
+Duende BFF with Blazor autorender                           duende-bff-blazor     [C#]      Web/Duende/BFF
+Duende IdentityServer Empty                                 duende-is-empty       [C#]      Web/Duende/IdentityServer
+Duende IdentityServer Quickstart UI (UI assets only)        duende-is-ui          [C#]      Web/IdentityServer
+Duende IdentityServer with ASP.NET Core Identity            duende-is-aspid       [C#]      Web/Duende/IdentityServer
+Duende IdentityServer with Entity Framework Stores          duende-is-ef          [C#]      Web/Duende/IdentityServer
+Duende IdentityServer with In-Memory Stores and Test Users  duende-is-inmem       [C#]      Web/Duende/IdentityServer
+Duende IdentityServer Quickstart                            duende-is-quickstart  [C#]      Web/Duende/IdentityServer
```

Comment:
Template name will be "Duende IdentityServer", with short name "duende-is".
 cc @StuFrankish